test/utils/DelegationHandler.sol

```diff
@@ -15,6 +15,10 @@ contract DelegationHandler is Test {
     address mockSecp256k1PublicKey = makeAddr("mockSecp256k1PublicKey");
     Key public mockSecp256k1Key = Key(0, KeyType.Secp256k1, true, abi.encodePacked(mockSecp256k1PublicKey));
 
+    address mockSecp256k1PublicKey2 = makeAddr("mockSecp256k1PublicKey2");
+    Key public mockSecp256k1Key2 =
+        Key(uint40(block.timestamp + 3600), KeyType.Secp256k1, false, abi.encodePacked(mockSecp256k1PublicKey2));
```

Comment:
nit: I worry that block.timestamp here could break tests if we do anything with vm.roll / warp in the future. Maybe just pin this to be safe?